lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -6,6 +6,15 @@ module Contrib
     # Rack module includes middlewares that are required to trace any framework
     # and application built on top of Rack.
     module Rack
+      # RACK headers to test when doing distributed tracing.
+      # They are slightly different from real headers as Rack uppercases everything
+
+      # Header used to transmit the trace ID.
+      HTTP_HEADER_TRACE_ID = 'HTTP_X_DATADOG_TRACE_ID'.freeze
+
+      # Header used to transmit the parent ID.
+      HTTP_HEADER_PARENT_ID = 'HTTP_X_DATADOG_PARENT_ID'.freeze
```

Comment:
OK @cabello we did discuss that point internally. At some point I had a test which was running this:

```ruby
# Below a quick example on how to hack and modify the standard headers.
Datadog::Monkey.without_warnings do # get rid of overriden constants warning
  module Datadog
    module Ext
      module DistributedTracing
        # These values can be used to communicate trace relationships
        # across multiple languages, so it's not recommended to modify
        # them, but should you want to do it, you can override them like this.
        HTTP_HEADER_TRACE_ID = 'x-custom-distributed-tracing-trace-id'.freeze
        HTTP_HEADER_PARENT_ID = 'x-custom-distributed-tracing-parent-id'.freeze
      end
    end
    module Contrib
      module Rack
        # Note the fact these are:
        # 1 - uppercased
        # 2 - '-' is replaced by '_'
        HTTP_HEADER_TRACE_ID = 'HTTP_X_CUSTOM_DISTRIBUTED_TRACING_TRACE_ID'.freeze
        HTTP_HEADER_PARENT_ID = 'HTTP_X_CUSTOM_DISTRIBUTED_TRACING_PARENT_ID'.freeze
      end
    end
  end
end
```
And it really does work, passes on CI fine, and using it on a toy app shows it works. You may use this. But I'm going to pull it from the code base, as we *really* don't want those to change, they should be shared among all languages indeed.